server.js

```diff
@@ -1,32 +1,169 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import theOfficeData from "./data/the_office_series.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
+// This is the database collection: 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+
+// Set up code according to documentation:
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
+const Episode = mongoose.model("Episode", {
+  Id: Number,
+  Season: Number,
+  EpisodeTitle: String,
+  About: String,
+  Ratings: Number,
+  Votes: Number,
+  Viewership: Number,
+  Duration: Number,
+  Date: String,
+  GuestStars: String,
+  Director: String,
+  Writers: String,
+});
+
+// Generate entries into the db from the json file:
+if (process.env.RESET_DB) {
+  const resetDataBase = async () => {
+    await Episode.deleteMany();
+    theOfficeData.forEach((singleEpisode) => {
+      const newEpisode = new Episode(singleEpisode);
+      newEpisode.save();
+    })
+  }
+  resetDataBase();
+}
+
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
+const listEndpoints = require('express-list-endpoints');
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Middleware that happens around each request, checks if db is connected
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({error: 'Service unavailable'})
+  }
+})
+
+// Route for start page
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  
+  // Lists the enpoints available:
+  res.json(listEndpoints(app));
```

Comment:
Great to have all of the endpoints collected here, makes it very easy to try them!